javascript/node/selenium-webdriver/chrome.js

```diff
@@ -224,6 +225,10 @@ function configureExecutor(executor) {
       Command.STOP_CASTING,
       'POST',
       '/session/:sessionId/goog/cast/stop_casting');
+  executor.defineCommand(
+      Command.SEND_DEVTOOLS_COMMAND_AND_GET_RETURN,
+      'POST',
+      '/session/:sessionId/goog/cdp/execute');
```

Comment:
Is this a replacement for `/session/:sessionId/chromium/send_command` ?

Would be better to just update the endpoint used by `sendDevToolsCommand()` than to add a new method to the API

Reply:
Very good point. I decided to make this a separate command to stay consistent with Chromium code which offers both endpoint to:

- only send a command - [send_command](https://chromium.googlesource.com/chromium/src.git/+/178d8ad3a5ac163aedd278b1721c4f31f9923bba/chrome/test/chromedriver/server/http_handler.cc#562)
- send the command and return the result [send_command_and_get_result](https://chromium.googlesource.com/chromium/src.git/+/178d8ad3a5ac163aedd278b1721c4f31f9923bba/chrome/test/chromedriver/server/http_handler.cc#568)

I haven't look how these different endpoints are implemented being the hood, but felt it would be more flexible to add the new functionality instead of replacing the send only one. This is debatable, though.